File: 1° semestre/APC verão/Raylan da Silva Sales_lab_3/C/Lab3a.c
#include<stdio.h>
#include<stdlib.h>
#define MAX 10
double fneuronio(double *ENTRADAS,double *PESOS,int TAM,double T){
    int i;
	double SOMAP=0;
	for(i=0;i<MAX;i++){
		SOMAP += *(ENTRADAS+i) * *(PESOS+i);
	}
	if(SOMAP > T)
		return 1;
	else
		return 0;
}
int main(){
	int i;
	double ENTRADAS[MAX],PESOS[MAX],T,resultado;
		for(i=0;i<MAX;i++){
			printf("Digite um valor de entrada:\n");
			scanf("%lf",&ENTRADAS[i]);
			printf("Digite um valor de peso:\n");
			scanf("%lf",&PESOS[i]);
		}
	printf("Digite o valor de T:\n");
	scanf("%lf",&T);

	resultado = fneuronio(ENTRADAS,PESOS,MAX,T);
	if(resultado == 1){
		printf("Neuronio ativado!");
	}else
		printf("Neuronio inibido!");
	return 0;
}

File: 1° semestre/APC verão/Raylan da Silva Sales_lab_3/Python/Lab3a.py
ENTRADAS = 10
PESOS = 10
for  i in range(0, 10):
    ENTRADA = float (input("Digite um valor de entrada: "))
    PESOS = float(input("Digite o valor do peso: "))
T = float(input("Digite o valor de T: "))

def fneuronio(ENTRADAS, PESOS, T):
    SOMAP = 0
    for i in range(0, 10):
        SOMAP += (ENTRADAS+1) * (PESOS+1)
    if SOMAP > T:
        return 1
    else:
        return 0

resultado = fneuronio(ENTRADAS,PESOS,T)
if(resultado == 1):
    print('Neuronio ativado!')
else:
    print('Neuronio inibido!')    
